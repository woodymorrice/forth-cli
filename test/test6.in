false if ."FAIL" else ."PASS" then
.
.