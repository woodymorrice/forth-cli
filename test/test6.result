2c2
< FAIL
---
> stack empty
