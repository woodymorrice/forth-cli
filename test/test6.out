PASS
stack empty
stack empty
